_resources/port1.0/group/rust-1.0.tcl

```diff
@@ -243,17 +243,17 @@ proc rust.add_bootstrap_components {architectures {components {rust-std rustc ca
                             sha256  e8f67154ae8faefb81be06b8804658cdde39d96e8fbd6b5240a9ef9d95f49f57 \
                             size    8607683 \
                             rust-std-${version_m1}-aarch64-apple-darwin${extract.suffix} \
```

Comment:
Because we need to pin older macOS versions to older Rust versions, I think that we should *probably* call out these versions explicitly. That means that if a bootstrap compiler needs to be downloaded, it’s *explicitly* supported and the older versions don't have those values missing.

Yes, it means that there's some carried along checksums, but I think it ensures compatibility over the long term.